src/gui/newwizard/states/serverurlsetupwizardstate.cpp

```diff
@@ -105,49 +106,66 @@ void ServerUrlSetupWizardState::evaluatePage()
         // and certificates are deleted from the access manager
         _context->resetAccessManager();
 
-        // first, we must resolve the actual server URL
-        auto resolveJob = Jobs::ResolveUrlJobFactory(_context->accessManager()).startJob(serverUrl, this);
+        // since classic WebFinger is not enabled, we need to check whether modern (oCIS) WebFinger is available
+        // therefore, we run the corresponding discovery job
+        auto checkWebFingerAuthJob = Jobs::DiscoverWebFingerServiceJobFactory(_context->accessManager()).startJob(serverUrl, this);
 
-        connect(resolveJob, &CoreJob::finished, resolveJob, [this, resolveJob]() {
-            resolveJob->deleteLater();
+        connect(checkWebFingerAuthJob, &CoreJob::finished, this, [job = checkWebFingerAuthJob, serverUrl, this]() {
+            // in case any kind of error occurs, we assume the WebFinger service is not available, and we continue the classic way
```

Comment:
```suggestion
            // in case any kind of error occurs, we assume the WebFinger service is not available, and we continue the default protocol
```

nothing classic about not using webfinger